gst/tensor_decoder/CMakeLists.txt

```diff
@@ -1 +1,2 @@
-ADD_LIBRARY(tensor_decoderOBJ OBJECT tensordec.c)
+set(DECODER_SOURCE tensordec.c tensordec_bounding_boxes_core.cc)
```

Comment:
Please come back to C (C89). Core code needs to follow C89 to be upstreamed to gst-plugins-good in the future. (or use Rust...)